src/backend/profile/profile.service.ts

```diff
@@ -154,7 +155,13 @@ export async function updateUserData(
     },
     data: userDataToUpdate,
   })
-
+  if (userDataToUpdate?.state) {
+    // I have no idea how to get current app url for link
+    discordNotification(
```

Comment:
**nitpick:** Let's change this func name to more specific, and action type, eg: `sendDiscordNotificationToWebhook`.